metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableMetricModule.java

```diff
@@ -87,6 +89,7 @@ public BigtableMetricModule(
         this.groups = groups.orElseGet(Groups::empty).or(DEFAULT_GROUP);
         this.project = project.orElseThrow(() -> new NullPointerException("project"));
         this.instance = instance.orElse(DEFAULT_INSTANCE);
+        this.profile = profile.orElse(null);
```

Comment:
should we have a default profile here instead of null?

Reply:
I thought about this but ended up using default without literally setting it up to "default". In BigtableConnectionBuilder it would behave the same way as before in case it is not set in config.